src/config/process-config.ts

```diff
@@ -1,16 +1,37 @@
+import fastGlob from 'fast-glob';
 import * as path from 'path';
-import { Config, ConfigFile } from './../types';
+import { Config, ConfigFile, ConfigRules, ConfigRulesFn } from './../types';
 
 export const getDirResolver = (filepath: string) => (dir: string) =>
     path.resolve(path.dirname(filepath), dir);
 
+const getRules = (
+    rulesFn: ConfigRulesFn,
+    root: string,
+    packages: string[] = ['*/**'],
+): ConfigRules => {
+    const pkgJsonGlobs = packages.map(p => `${p}/package.json`);
```

Comment:
I'm appending package.json to the package globs here but am not sure if we should be so opinionated. The main idea was that people could just import and re-use their existing packages list from their monorepo tooling, e.g. `workspaces.packages` field in package.json for yarn workspaces or `bolt.workspaces` for bolt without needing to append package.json to the glob themselves.

The reason why I've defaulted to searching for all package.json's instead of scoping it to fields specified by these tools (yarn workspaces/bolt/lerna) is that they vary in location and I'm not sure the tradeoff of maintaining these was worth it. Package.json's should be a good enough indicator of whether something is a package or not, without relying on these fields.